workspaces/marketplace/packages/cli/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "@red-hat-developer-hub/marketplace-cli",
+  "description": "CLI for the Marketplace",
+  "version": "0.0.1",
+  "backstage": {
+    "role": "cli"
+  },
+  "private": true,
+  "homepage": "https://red.ht/rhdh",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/redhat-developer/rhdh-plugins",
+    "directory": "workspaces/marketplace/packages/cli"
+  },
+  "keywords": [
+    "backstage"
+  ],
+  "license": "Apache-2.0",
+  "main": "dist/index.cjs.js",
+  "scripts": {
+    "build": "backstage-cli package build",
+    "lint": "backstage-cli package lint",
+    "test": "backstage-cli package test",
+    "clean": "backstage-cli package clean",
+    "start": "nodemon --"
+  },
+  "bin": "bin/marketplace-cli",
+  "files": [
+    "bin",
+    "dist/**/*.js"
+  ],
+  "dependencies": {
+    "@backstage/catalog-model": "^1.7.0",
+    "@backstage/cli": "^0.26.11",
+    "@backstage/cli-node": "^0.2.7",
+    "@backstage/errors": "^1.2.4",
+    "@manypkg/get-packages": "^2.2.1",
+    "@red-hat-developer-hub/backstage-plugin-marketplace-common": "workspace:^",
+    "chalk": "^4.1.2",
+    "commander": "^12.0.0",
+    "find-up": "^5.0.0",
+    "fs-extra": "^11.2.0",
+    "inquirer": "^8.0.0",
+    "replace-in-file": "^7.1.0",
+    "semver": "^7.6.0",
```

Comment:
Same here. Can you please double-check which dependencies are really needed for the cli.